circuitpython_library_pypi_stats.md

```diff
@@ -313,3 +312,12 @@
 | Adafruit CircuitPython BLE Beacon (adafruit-circuitpython-ble-beacon) | 113 downloads |
 | Adafruit CircuitPython PCF8575 (adafruit-circuitpython-pcf8575) | 62 downloads |
 | Adafruit CircuitPython ENS160 (adafruit-circuitpython-ens160) | 59 downloads |
+
+## Blinka
```

Comment:
This should be a 3-header (`###`) to match the header above the library table above.